core/node/node_framework/src/implementations/layers/base_token/base_token_ratio_persister.rs

```diff
@@ -81,37 +82,37 @@ impl WiringLayer for BaseTokenRatioPersisterLayer {
             .contracts_config
             .base_token_addr
             .expect("base token address is not set");
-        let diamond_proxy_contract_address = self.contracts_config.diamond_proxy_addr;
-        let chain_admin_contract_address = self.contracts_config.chain_admin_addr;
-        let token_multiplier_setter_wallet = self
-            .wallets_config
-            .token_multiplier_setter
-            .expect("base token adjuster wallet is not set")
-            .wallet;
 
-        let tms_private_key = token_multiplier_setter_wallet.private_key();
-        let tms_address = token_multiplier_setter_wallet.address();
-        let EthInterfaceResource(query_client) = input.eth_client;
+        let mut l1_params: Option<BaseTokenRatioPersisterL1Params> = None;
```

Comment:
you can avoid mutability with `let l1_params  = self.wallets_config.token_multiplier_setter.map(...)`

Reply:
thank you - fixed